package-lock.json

```diff
@@ -1,8 +1,5901 @@
 {
   "name": "cql-execution",
   "version": "2.4.2",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This does not make a functional difference to the commit.  I can revert if desired.

The note I would add, is that all versions of Node that came with npm v6 and earlier (lockfileVersion 1) have reached end-of-life except Node 14 which is in maintenance status until 2023-04-30 (https://github.com/nodejs/Release).

Reply:
Yeah, I agree that NPM 8 is the future (and much of the present too).  I just don't want the package-lock to change formats every time someone different pushes a PR.  @mgramigna - is Abacus using NPM 8 now on other projects?  If so, then let's switch cql-execution too (as @birick1 has done here).  If not, then let's not switch this yet -- but let's make plans for when it will happen.

Reply:
After discussion, we are totally good with npm 8 now. It's time